pages/blog/_posts/reimagining-information-architecture.mdx

```diff
@@ -0,0 +1,82 @@
+
+![](/assets/blog/reimagining-information-architecture/image4.png)
+
+## How the new Information Architecture simplifies navigation
+
+### Enhance the overall developer experience through an intuitive navigation system
+
+By organizing information into two primary interaction modes, Monitor and Manage, we have created a navigation system that aligns closely with the natural workflow of our users. This intuitive structure allows developers to quickly find and use the tools they need, enhancing their overall experience and making their interactions with Inngest more efficient and satisfying.
+
+### Reduce the learning curve for users by ensuring consistency across products
+
+Our new IA offers consistent navigation and layout across both the Dev Server and Cloud Application. By standardizing the navigation and layout, we ensure that users can easily transition between these two environments without needing to relearn the interface. This consistency significantly reduces the learning curve and helps users become proficient more quickly. Before, users had difficulty switching between the Dev Server and Cloud Application. Aligning the functionality, layout, and terminology between the Dev Server and the Cloud Application reduces the cognitive load and for users, making transitioning between the Dev Server and the Cloud Application a much more seamless experience.  
+![](/assets/blog/reimagining-information-architecture/image5.png)
+
+### Align the IA with mental models
+
+The Manage section is specifically designed to reflect the Inngest mental model, making it easier for users to understand and navigate the app. By aligning the IA with how users think about their systems, we improved usability and created a more intuitive experience that feels natural and logical to our users.   
+
+![](/assets/blog/reimagining-information-architecture/image6.png)
+
+For example, if users wanted to find their most recently failed runs in the Cloud Application, it would require them to dig into the functions page and manually sift through individual functions. Now users can go to the Runs page under manage and quickly sort by all failed runs
+
+### A future-proof navigation
+
+The new structure is designed to be scalable, allowing us to easily incorporate new features and functionalities in the future without major overhauls. This robust foundation ensures that our IA can grow and adapt as Inngest evolves, supporting both current needs and future developments.
+
+### Quick access to observability features
+
+The Monitor grouping offers tools for debugging, troubleshooting, and observing systems, providing users with the clarity they need to understand workflow and system performance. Global runs and our in-flight monitoring dashboard will be found under this section.
```

Comment:
Should we show our Dashboard design here, or at least mention it here in some more specific way?